main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: amplifier
+//  action 1: guitar is connected to the amplifier
+amplifier.guitarConnected();
+//  action 2: turn on overdrive
+amplifier.turnOnOverdrive();
+//  action 3: turn on delay effect
+amplifier.turnOnDelay();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: cat
+//  action 1: the cat is born
+cat.Born();
+//  action 2: cat eats some food
+cat.eatFood();
+//  action 3: cat is playing with the ball
+cat.playingWithBall();

```

Comment:
change your plain-english expression to not use `is`
`cat plays with the ball`
`cat.playsWithBall();`
- [ ] revise